addons/project/views/project_project_views.xml

```diff
@@ -76,7 +76,7 @@
                             <field name="date" invisible="1" />
                         </group>
                     </group>
-                    <notebook>
+                    <notebook class="overflow-hidden">
```

Comment:
I think we could do this instead: 

```diff
diff --git a/addons/project/static/src/css/project.css b/addons/project/static/src/css/project.css
index d7c8acf7273a..bb1e9a601756 100644
--- a/addons/project/static/src/css/project.css
+++ b/addons/project/static/src/css/project.css
@@ -10,5 +10,5 @@
 }

 .o_form_project_project  .o_setting_box {
-  margin-left: 1.5rem;
+  padding-left: 1.5rem;
 }
```

What do you think? 🙂

Reply:
@cariy-odoo @kcv-odoo ping 🙂

Reply:
Done! Thank you :innocent: